update-dois/src/main/java/org/reactome/release/updateDOIs/findNewDOIsAndUpdate.java

```diff
@@ -29,42 +33,89 @@ public void setGkCentralAdaptor(MySQLAdaptor adaptor) {
     this.dbaGkCentral = adaptor;
   }
 
-  @SuppressWarnings("unchecked")
-public void findNewDOIsAndUpdate(long authorId) {
-	  
+  @SuppressWarnings({"unchecked","rawtypes"})
+  public void findAndUpdateDOIs(long authorIdTR, long authorIdGK, String pathToReport) {
+
     Collection<GKInstance> dois;
     Collection<GKInstance> gkdois;
-    
-//	String creatorFile = "org.reactome.release.updateDOIs.UpdateDOIs";
-//	GKInstance instanceEditTestReactome = this.createInstanceEdit(authorId, creatorFile);
-//	GKInstance instanceEditGkCentral = this.createInstanceEdit(authorId, creatorFile);
-	
+
+	String creatorFile = "org.reactome.release.updateDOIs.UpdateDOIs";
+	GKInstance instanceEditTestReactome = this.createInstanceEdit(this.dbaTestReactome, authorIdTR, creatorFile);
+	GKInstance instanceEditGkCentral = this.createInstanceEdit(this.dbaGkCentral, authorIdGK, creatorFile);
+
+    HashMap reportContents = this.getReportContents(pathToReport);
+    int reportHits = 0;
+    int fetchHits = 0;
+    ArrayList updatedDOIs = new ArrayList();
     try {
-     dois = this.dbaTestReactome.fetchInstanceByAttribute("Pathway", "doi", "NOT REGEXP", "^10.3180");
-//     dois = this.dbaTestReactome.fetchInstanceByAttribute("Pathway", "DB_ID", "REGEXP", "8939211|9006115|9018678|9033241");
+
+      dois = this.dbaTestReactome.fetchInstanceByAttribute("Pathway", "doi", "NOT REGEXP", "^10.3180");
+      dois = this.dbaTestReactome.fetchInstanceByAttribute("Pathway", "DB_ID", "REGEXP", "8939211|9006115|9018678|9033241");
```

Comment:
`dois` gets a value assigned twice. I think the second assignment will overwrite the results of the first one, so why is the first one done?

Reply:
Second one should have been commented out or removed, fixed